htdocs/recruitment/recruitmentcandidature_card.php

```diff
@@ -257,13 +257,13 @@
 $formfile = new FormFile($db);
 $formproject = new FormProjets($db);
 
-$title = $langs->trans("RecruitmentCandidature");
-$help_url = '';
-llxHeader('', $title, $help_url);
-
 
 // Part to create
 if ($action == 'create') {
+	$title = $langs->trans('NewCandidature');
+	$help_url = '';
+	llxHeader('', $title, $help_url);
```

Comment:
The llxHeader must always be at top (so line 260) because it contains a lot of security features like adding HTTP headers and we must be sure that it is executed for all "if" cases.
A lot of security bug were introduces with previous moves of llxHeader and i started to revert them so we always have a llxHeader at begin of the "/* View" part of a page. This avoid to forgot it in some if cases and avoid to have to repeat it for each if later.

If the title change, we should add the if line 260.

Reply:
did move the function up to the top code level.